Xamarin.Essentials/FilePicker/FilePicker.android.cs

```diff
@@ -11,10 +11,6 @@ public static partial class FilePicker
     {
         static async Task<IEnumerable<FileResult>> PlatformPickAsync(PickOptions options, bool allowMultiple = false)
         {
-            // we only need the permission when accessing the file, but it's more natural
-            // to ask the user first, then show the picker.
-            await Permissions.EnsureGrantedAsync<Permissions.StorageRead>();
```

Comment:
I'm not too familiar with this area, but would this permission check still be needed for Android versions prior to 30? ie. should there be a version check here?

Reply:
From what Google is reporting when people submit apps is that they will reject them even if they have the setting in there. I tested it out on older and newer devices and it seemed to work